app/controllers/static_pages_controller.rb

```diff
@@ -0,0 +1,4 @@
+class StaticPagesController < ApplicationController
+    def home; end    
+    def help; end
+  end
```

Comment:
![Uploading image.png…]()

Reply:
sorry, the comment says "Uploading image.png…" . I cannot see what need to be fix

Reply:
done